pensieve/config.py

```diff
@@ -293,6 +438,7 @@ class AnalysisSpec:
 
     experiment: ExperimentSpec = attr.Factory(ExperimentSpec)
     metrics: MetricsSpec = attr.Factory(MetricsSpec)
+    statistics: StatisticSpec = attr.Factory(StatisticSpec)
```

Comment:
I think statistics need to be specified in the context of a metric, not at the top level.